x/concentrated-liquidity/incentives.go

```diff
@@ -928,6 +932,7 @@ func (k Keeper) claimAllIncentivesForPosition(ctx sdk.Context, positionId uint64
 	// Loop through each uptime accumulator for the pool.
 	for uptimeIndex, uptimeAccum := range uptimeAccumulators {
 		// Check if the accumulator contains the position.
+		// @Dev: There should never be a case where you can have a position for 1 accumulator, and not the rest right?
```

Comment:
This is the case now that we handle early exits with forfeiting logic (back when we did it based on freezing, this claim did not hold)

Reply:
Ah I see